{"hierarchy":{"paths":[["doc:\/\/com.DocuSign.DocuSIgnAPI\/documentation\/DocuSignAPI","doc:\/\/com.DocuSign.DocuSIgnAPI\/documentation\/DocuSignAPI\/DSAPIConnectFailureResult"]]},"identifier":{"url":"doc:\/\/com.DocuSign.DocuSIgnAPI\/documentation\/DocuSignAPI\/DSAPIConnectFailureResult\/statusMessage","interfaceLanguage":"swift"},"sections":[],"abstract":[{"text":"A human-readable message describing the connection status.","type":"text"}],"primaryContentSections":[{"declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"text":"statusMessage","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"kind":"text","text":"?"}],"platforms":["iOS"],"languages":["swift"]}],"kind":"declarations"}],"schemaVersion":{"major":0,"patch":0,"minor":3},"metadata":{"modules":[{"name":"DocuSignAPI"}],"symbolKind":"property","externalID":"s:11DocuSignAPI25DSAPIConnectFailureResultV13statusMessageSSSgvp","role":"symbol","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"statusMessage","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"roleHeading":"Instance Property","title":"statusMessage"},"variants":[{"paths":["\/documentation\/docusignapi\/dsapiconnectfailureresult\/statusmessage"],"traits":[{"interfaceLanguage":"swift"}]}],"kind":"symbol","references":{"doc://com.DocuSign.DocuSIgnAPI/documentation/DocuSignAPI/DSAPIConnectFailureResult/statusMessage":{"kind":"symbol","title":"statusMessage","type":"topic","abstract":[{"type":"text","text":"A human-readable message describing the connection status."}],"role":"symbol","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"statusMessage"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"url":"\/documentation\/docusignapi\/dsapiconnectfailureresult\/statusmessage","identifier":"doc:\/\/com.DocuSign.DocuSIgnAPI\/documentation\/DocuSignAPI\/DSAPIConnectFailureResult\/statusMessage"},"doc://com.DocuSign.DocuSIgnAPI/documentation/DocuSignAPI":{"kind":"symbol","type":"topic","role":"collection","url":"\/documentation\/docusignapi","abstract":[],"title":"DocuSignAPI","identifier":"doc:\/\/com.DocuSign.DocuSIgnAPI\/documentation\/DocuSignAPI"},"doc://com.DocuSign.DocuSIgnAPI/documentation/DocuSignAPI/DSAPIConnectFailureResult":{"kind":"symbol","role":"symbol","url":"\/documentation\/docusignapi\/dsapiconnectfailureresult","type":"topic","navigatorTitle":[{"kind":"identifier","text":"DSAPIConnectFailureResult"}],"fragments":[{"text":"struct","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"DSAPIConnectFailureResult"}],"title":"DSAPIConnectFailureResult","identifier":"doc:\/\/com.DocuSign.DocuSIgnAPI\/documentation\/DocuSignAPI\/DSAPIConnectFailureResult","abstract":[{"text":"This object contains details about a Connect failure result.","type":"text"}]}}}